pcsx2/CDVD/IsoFS/IsoFS.cpp

```diff
@@ -49,8 +50,9 @@ IsoDirectory::IsoDirectory(SectorSource& r)
 	uint i = 16;
 
 	m_fstype = FStype_ISO9660;
-
-	while( !done )
+	
+	bool loadFromDisc = g_Conf->CdvdSource != CDVD_SourceType::NoDisc;
+	while( !done && loadFromDisc)
```

Comment:
Huh didn't realize I left the else in there. I'm pretty sure I had this working at some point, but I'll have to look into it or remove this completely.